src/Microsoft.ML.Transforms/UngroupTransform.cs

```diff
@@ -79,6 +79,18 @@ public sealed class Arguments : TransformInputBase
 
         private readonly SchemaImpl _schemaImpl;
 
+        /// <summary>
+        /// Convenience constructor for public facing API.
```

Comment:
>Convenience constructor for public facing API. [](start = 12, length = 46)

This is not a really great description.

Reply:
I was thinking to open a separate issue to include clear description for each convenience constructors. What do you say?

---
In reply to: [201912748](https://github.com/dotnet/machinelearning/pull/520#discussion_r201912748) [](ancestors = 201912748,201912669,201910848)

Reply:
#524

---
In reply to: [202117587](https://github.com/dotnet/machinelearning/pull/520#discussion_r202117587) [](ancestors = 202117587,201912748,201912669,201910848)

Reply:
Fair enough, sounds good @zeahmed

---
In reply to: [202121318](https://github.com/dotnet/machinelearning/pull/520#discussion_r202121318) [](ancestors = 202121318,202117587,201912748,201912669,201910848)